internal/server/db/operationtype/operation_type.go

```diff
@@ -75,6 +75,10 @@ const (
 	RenewServerCertificate
 	RemoveExpiredTokens
 	ClusterHeal
+	BucketBackupCreate
```

Comment:
Can you put those changes into a new `incusd/db/operation: Add storage volume backup types` commit?